issue.go

```diff
@@ -608,6 +608,27 @@ func (s *IssueService) Update(issue *Issue) (*Issue, *Response, error) {
 	return &ret, resp, nil
 }
 
+// Update updates an issue from a JSON representation. The issue is found by key.
+//
+// https://docs.atlassian.com/jira/REST/7.4.0/#api/2/issue-editIssue
+func (s *IssueService) UpdateIssue(jiraId string, data map[string]interface{}) (*Response, error) {
+	apiEndpoint := fmt.Sprintf("rest/api/2/issue/%v", jiraId)
+	req, err := s.client.NewRequest("PUT", apiEndpoint, data)
+	fmt.Println(apiEndpoint, req, err)
```

Comment:
Please remove this side effect here. Every tool that uses the message would have an output on stdout/stderr with this.